src/redux/books/books.js

```diff
@@ -0,0 +1,26 @@
+// books.js
+import { combineReducers } from 'redux';
+// Actions
```

Comment:
- [ ] You should import `combineReducer` in `configureStore.js` so that you can use it to combine other reducers later on.